ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.7-periodics.yaml

```diff
@@ -11,21 +11,10 @@ periodics:
     path_alias: knative.dev/serving
     repo: serving
   labels:
-    ci-operator.openshift.io/variant: "411"
+    ci-operator.openshift.io/variant: "412"
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-knative-serving-release-v1.7-411-e2e-aws-ocp-411-continuous
-  reporter_config:
```

Comment:
The slack notifications are removed here. Is this intentional? Sometimes they have to be manually added back after running "make jobs"